detectron2/modeling/proposal_generator/proposal_utils.py

```diff
@@ -100,7 +97,7 @@ def find_top_rpn_proposals(
         boxes.clip(image_size)
 
         # filter empty boxes
-        keep = boxes.nonempty(threshold=min_box_size)
+        keep = boxes.nonempty(threshold=float(min_box_size))
```

Comment:
This change can be removed if we fix the signature of `find_top_rpn_proposals` to use `float`, and add `float(min_box_size)` in `RPN.__init__`

Reply:
done. I also change `_C.MODEL.PROPOSAL_GENERATOR.MIN_SIZE` in `detectron2/config/defaults.py` from 0 to 0.0 which make it more clear for users.